content/en/citing-numpy.md

```diff
@@ -0,0 +1,28 @@
+**Citing NumPy**
+
+If NumPy has been significant in your research, and you would like to acknowledge the project in your academic publication, we suggest citing the following papers:
+
+
+
+*   Travis E, Oliphant. A guide to NumPy, USA: Trelgol Publishing, (2006).
+*   Stéfan van der Walt, S. Chris Colbert and Gaël Varoquaux. The NumPy Array: A Structure for Efficient Numerical Computation, Computing in Science & Engineering, 13, 22-30 (2011),[ DOI:10.1109/MCSE.2011.37](http://dx.doi.org/10.1109/MCSE.2011.37) ([publisher link](http://scitation.aip.org/content/aip/journal/cise/13/2/10.1109/MCSE.2011.37))
+
+_In BibTeX format:_
+
```

Comment:
This is missing the first line: `@article{van2011numpy,`

It also doesn't render well like this. Instead try to use triple backticks above and below this:
```
@article{van2011numpy,
  title={The NumPy array: a structure for efficient numerical computation},
  author={Van Der Walt, Stefan and Colbert, S Chris and Varoquaux, Gael},
  journal={Computing in Science \& Engineering},
  volume={13},
  number={2},
  pages={22},
  year={2011},
  publisher={IEEE Computer Society}
}
```